lib/sales_engine.rb

```diff
@@ -153,56 +145,68 @@ def find_invoice_from_transaction(id)
     invoice_repository.find_by_id(id)
   end
 
-
-  def find_successful_invoices_from_merchant(id, date)
+  def find_invoice_ids_by_merchant_and_date(merchant_id, date)
```

Comment:
Unused method argument - `merchant_id`. If it's necessary, use `_` or `_merchant_id` as an argument name to indicate that it won't be used.